docs/contributors/native-mobile.md

```diff
@@ -0,0 +1,6 @@
+# React Native based mobile Gutenberg
+Intertwined with the web codepaths, the Gutenberg repo also includes the React Native based mobile tree. The mobile client is packaged and released via the [official WordPress apps](https://wordpress.org/mobile/). Even though the build pipeline is rather different and lives in its own repo for now ([here's the native mobile repo](https://github.com/wordpress-mobile/gutenberg-mobile)), the source code itself is taken directly from this repo and the "web" side codepaths.
```

Comment:
Could you also wrap `React Native` with a link for people who don't know what it is? :)

Reply:
Makes sense, addressed with 3c62db5.